recipes/calour/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "calour" %}
+{% set version = "2018.05" %}
+{% set sha256 = "09b62a876e9bf32db996589362219b622feb0ca7b3a0739531e80fb2ae2b9246" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - matplotlib >=2.0.0
+    - scikit-learn >=0.19.1
+    - scikit-bio >=0.5.2
+    - biom-format
```

Comment:
It looks like you're going to need to have `biom-format` submitted and accepted before you can build this package.

Reply:
it's already there: https://bioconda.github.io/recipes/biom-format/README.html

Reply:
That looks like it's hosted on `bioconda`. My understanding from looking at the config files on conda-forge is that it only looks for packages in `defaults`, `conda-forge` and `local` channels.

Reply:
ah, sorry. you're right. will do that